sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -0,0 +1,181 @@
+} from "@azure/ai-language-textauthoring";
+import { BlobServiceClient, ContainerClient } from "@azure/storage-blob";
+import { DefaultAzureCredential } from "@azure/identity";
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+import * as fs from "fs";
+import path from "path";
+import decompress from "decompress";
+import {
+  customEntityAssets,
+  customMultiLabelAssets,
+  customSingleLabelAssets,
+} from "../customTestsAssets";
+dotenv.config();
+
+const storageInputContainerName = "documents";
+const language = "en-us";
+const trainingConfigVersion = "2022-05-01";
+
+async function createStorageBlob(): Promise<ContainerClient> {
+  /* Return a container client */
+  const defaultAzureCredential = new DefaultAzureCredential();
+  // Get documents containers
+  const blobServiceClient = new BlobServiceClient(
+    process.env["STORAGE_ENDPOINT"] || "string",
+    defaultAzureCredential
+  );
+  return blobServiceClient.getContainerClient(storageInputContainerName);
+}
+
+async function uploadDocuments(fileName: string): Promise<void> {
+  const client = await createStorageBlob();
+
+  // Unpack zip files and upload to blob
+  await decompress(path.join(__dirname, `${fileName}.zip`), __dirname);
+  const dir = path.join(__dirname, fileName);
+  fs.readdir(dir, function (err, files) {
+    if (err) {
+      console.log("Error while listing", err);
+      return;
+    }
+    files.forEach(async function (fileName) {
+      if (fileName.endsWith(".txt")) {
+        let blob = client.getBlockBlobClient(fileName);
+        let filePath = path.join(dir, fileName);
+        await blob.uploadFile(filePath);
+      }
+    });
+  });
+}
+
+async function polling(client: TextAuthoringClient, project: any) {
+  try {
+    const poller = getLongRunningPoller(client, project);
+    await poller.pollUntilDone();
+  } catch (error) {
+    console.log("Poller encountered error", error);
+  }
+}
+
+export async function createCustomTestProject(
+  client: TextAuthoringClient,
+  projectKind:
+    | "CustomEntityRecognition"
+    | "CustomSingleLabelClassification"
+    | "CustomMultiLabelClassification"
+    | "CustomAbstractiveSummarization",
+  projectName: string,
+  deploymentName: string
+) {
+  let files: string;
+  let assets:
+    | ExportedCustomSingleLabelClassificationProjectAssets
+    | ExportedCustomMultiLabelClassificationProjectAssets
+    | ExportedCustomEntityRecognitionProjectAssets;
+  switch (projectKind) {
+    case "CustomSingleLabelClassification":
+      assets = customSingleLabelAssets;
+      files = "WebOfScience";
+      break;
+
+    case "CustomMultiLabelClassification":
+      assets = customMultiLabelAssets;
+      files = "";
+      break;
+
+    case "CustomEntityRecognition":
+      assets = customEntityAssets;
+      files = "LoanAgreements";
+      break;
+
+    default:
+      assets = { projectKind: "CustomSingleLabelClassification" };
+      files = "";
+      break;
```

Comment:
could we just remove the custom abstractive summarization kind and remove this default clause?
```suggestion
```